### DISCUSSION TITLE
Video encode pause if vidcoder cannot delete file

### DISCUSSION THREAD

**Discussion by voska7** [RandomEngy/VidCoder#1021] [Ideas]
Video encode pause if vidcoder cannot delete file for batch encode. like if file is using by some other process and vidcoder cannot delete the file it give popup box error. until we close the error vidcoder pause the encode. will it be possible not to pause and encode the rest of the video file?

**Comment by RandomEngy** [comment]
This will be changed to a non-blocking log message in the next Beta release.

**Reply by RandomEngy** [reply]
Change is in 8.10 Beta
